lib/rpmsg/rpmsg_virtio.c

```diff
@@ -414,6 +449,33 @@ static int rpmsg_virtio_send_offchannel_nocopy(struct rpmsg_device *rdev,
 	return len;
 }
 
+static int rpmsg_virtio_release_tx_buffer(struct rpmsg_device *rdev, void *rxbuf)
+{
+	struct rpmsg_virtio_device *rvdev;
+	struct rpmsg_hdr *rp_hdr;
+	struct vbuff_reclaimer_t *r_desc = rxbuf;
```

Comment:
it's better to let r_desc point to rp_hdr and remove addr field from vbuff_reclaimer_t